src/vstart.sh

```diff
@@ -757,6 +769,13 @@ EOF
 		ln -s $kstore_path $CEPH_DEV_DIR/osd$osd
 	    else
 		mkdir -p $CEPH_DEV_DIR/osd$osd
+		if [ -n "${bluestore_dev[$osd]}" ]; then
+		    dd if=/dev/zero of=${bluestore_dev[$osd]} bs=1M count=1
```

Comment:
`wipefs -a "${bluestore_dev{$osd}}"` would be better I think? That also calls `ioctl` to reload the partition tables.

Reply:
Good idea.

Reply:
Hmm, except that it doesn't seem to actually work.

This command fails if I use wipefs instead of just dd'ing over the first part of the device. If I run wipefs vs the device after creating a bluestore OSD there, a subsequent bluestore mkfs fails.

I think I'll stick with dd, at least for now.

Reply:
Maybe bluestore is being too clever and looking beyond the parition table? You could do both: (a) wipefs to clear and reload the partition table and (b) dd to wipe out any bluestore headers.